test/money_test.js

```diff
@@ -21,6 +21,12 @@ describe("Money", function() {
       expect(money.cents).to.be(100);
     });
 
+    it("returns correct money string value", function() {
+      const moneyString = "10.2";
```

Comment:
I'd also add the weird case of `10.200` which was coming out as `12`

Reply:
I added a test for that in the last branch